.github/workflows/tasklist-docker-tests.yml

```diff
@@ -98,9 +98,6 @@ jobs:
           context: .
           push: true
           tags: ${{ env.TASKLIST_TEST_DOCKER_IMAGE }}
-          cache-from: type=gha,ignore-error=true
-          cache-to: type=gha,mode=max,ignore-error=true
-          no-cache-filters: build,distball,dist
```

Comment:
❓ So not configuring disables the cache?

Reply:
It disables exporting the cache to an external service (S3, GHA, ...) is my understanding from https://github.com/moby/buildkit/?tab=readme-ov-file#cache - Docker will still have its layer cache file structure on disk but not upload that stuff anywhere when the build ends.